src/ec/curve25519/ops.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 Brian Smith.
```

Comment:
I suggest checking the other `curve25519` modules while you're in there— the top-level `curve25519` module date should probably be changed.